addons/mail/static/src/js/thread.js

```diff
@@ -112,27 +112,77 @@ var Thread = Widget.extend({
         _.each(msgs, function(msg) {
             self.$('[data-message-id="' + msg.id + '"] .o_mail_timestamp')
                 .data('date', msg.date);
+
+            var $msg = self.$('.o_thread_message[data-message-id="'+ msg.id +'"]');
+            self.insert_read_more($msg);
+        });
+
+        if (!this.update_timestamps_interval) {
+            this.update_timestamps_interval = setInterval(function() {
+                self.update_timestamps();
+            }, 1000*60);
+        }
+    },
+
+    insert_read_more: function ($element) {
+
+        var self = this;
+
+        var groups = [];
+        var current_group;
```

Comment:
Each group is a "read more/read less" element with nodes to toggle (current_group).
How to rename it?
groups = read_more_items ?
current_group = node_to_toggle ?